Using_Crypt.md

```diff
@@ -1,13 +1,15 @@
-## Using CRYPT with the Vault
+## Using CRYPT With the Vault
 
-A new feature was added to the Vault which allows users to utilize encrypted passwords in configuration files which are **not** stored in the vault.
+A new feature was added to the Vault which allows users to utilize encrypted values in configuration files which are **not** stored in the vault.
 
-### Installation
+### Installing the Vault
 
-See the [INSTALL](./INSTALL.md) file for instructions on installation and usage of the vault (minus the CRYPT feature).
+See the [INSTALL](./INSTALL.md) file for instructions on installation and usage of the vault (minus the CRYPT feature, which is fully documented here).
 
 ### Using the CRYPT feature
 
+#### Configuring Tomcat
+
 Configure Tomcat to use your password from an encrypted or plain text string:
 
 - **Encrypted**: Add the encryption password to the vault and put the VAULT reference in your vault.properties as follows:
```

Comment:
This is not a part of this PR. But I think it would be better to provide a concrete example command for this part. For example:

Add the encryption password to the vault store:

~~~
$ bin/vault.sh --keystore /path/to/vault.keystore --keystore-password my_password123 --alias my_vault --enc-dir /path/to/vault/encryption_dir/ --iteration 120 --salt 1234abcd --vault-block my_block --attribute my_encryption_password --sec-attr MyEncryptionPassword
~~~

then, put the VAULT reference in your vault.properties as follows:

~~~
ENCRYPTION_PASSWORD=VAULT::my_block::my_encryption_password::
~~~

Reply:
Fixed.